### ISSUE TITLE
UI Component Library: Checkbox/Radiobutton Accessibility Violations

### ISSUE THREAD

**Issue by kcullion** [icgc-argo/argo-platform#175]
Please fix the accessibility violations in 
- [x] Checkbox
- [x] Radio
- [x] FormCheckbox

**Comment by rosibaj** [comment]
closing in sprint as review will be merged today
